activitypub.php

```diff
@@ -81,6 +81,7 @@ function plugin_init() {
 	\add_action( 'init', array( __NAMESPACE__ . '\Health_Check', 'init' ) );
 	\add_action( 'init', array( __NAMESPACE__ . '\Scheduler', 'init' ) );
 	\add_action( 'init', array( __NAMESPACE__ . '\Comment', 'init' ) );
+	\add_action( 'init', array( __NAMESPACE__ . '\Collection\Extra_Fields', 'init' ) );
```

Comment:
I would love to avoid hooks in the collections. It's true that there is no real consistent way how classes with hooks are used, maybe we should discuss that on monday.

Reply:
Yeah great topic, let's dig into it.